src/code/z_demo.c

```diff
@@ -586,19 +613,18 @@ void Cutscene_TerminatorImpl(PlayState* play, CutsceneContext* csCtx, CsCmdBase*
     }
 }
 
-// Command 0x15E
-void Cutscene_Command_Terminator(PlayState* play, CutsceneContext* csCtx, CsCmdBase* cmd) {
-    if (cmd->base == 1) {
-        if (csCtx->frames == cmd->startFrame) {
-            Cutscene_TerminatorImpl(play, csCtx, cmd);
+void CutsceneCmd_Destination(PlayState* play, CutsceneContext* csCtx, CsCmdDestination* cmd) {
+    if (cmd->type == CS_DESTINATION_DEFAULT) {
+        if (csCtx->curFrame == cmd->startFrame) {
```

Comment:
so the term "destination" was created specifically to represent the individual blocks of code that send you somewhere else. thankfully in MM this is mostly refactored out, in favor of a simpler entrance index included in the cs script list. 
I was feeling like MM would probably just name this more after that it does, which is send you to a new entrance, instead of having the "destination" terminology, since it doesnt have these custom code blocks anymore

That is, except for the boss warps. these are more in line with the oot "destination", but they are now a small percentage of what the command handles.

idk, consistency with oot is probably preferable in the end, but wanted to voice my thoughts that what is meant by the term "destination" in oot matters alot less now

Reply:
That's fair, I didn't follow the discussion of the term "destination" closely, but that term seemed to make sense to me that it's sending you to an entrance, so even with the reworked system, the term still made sense in my mind as appropriate. This one definitely doesn't need to be tied to OoT, but `CutsceneCmd_Destination` still makes sense to me at least

Reply:
yeah, im ok with letting the term exist for the sake of cross game sync. 

But yeah, if oot had the entrance index mechanic and didnt have the horrible switch that it does, the term destination probably wouldnt have been created and it'd just be "scene transition" or "next entrance" or something lol

Reply:
OoT would always have the "horrible switch" because the whole point of it is to be able to trigger events and customize the transition into the next scene, aside from the ganon barrier case.

Reply:
a very large percentage of them could be handled the exact same way mm does it. custom cases certainly do exist but they arent common